### ISSUE TITLE
Layouts dropdown cutoff if the Flexible field is placed too high in a panel.

### ISSUE THREAD

**Issue by gluebag** [whitecube/nova-flexible-content#12]
I noticed that if you have too many layouts, and place the Flexible field close to the top of your resources' $fields, the dropdown when you go to add a new group gets cut off by the container.

<img width="1027" alt="Screen Shot 2019-03-12 at 4 33 31 PM" src="https://user-images.githubusercontent.com/4186151/54234250-20342a80-44e5-11e9-98cb-e0f3aa507809.png">

**Comment by voidgraphics** [comment]
Good catch, thanks. We'd accept a PR that fixes this if anyone is up to it.

**Comment by Nyratas** [comment]
The problem comes from a `overflow: hidden` on nova's fields container. I see two possible fixes:

- Move the dropdown DOM element outside the problematic divd position it absolutlely ;
- Show the dropdown below the button if we don't have enough space above.

The first one may be a little bit tricky with Vue.js I think, the second one doesn't solve anything if the Flexible field is the only field in the form.

Other ideas ?

**Comment by voidgraphics** [comment]
Maybe we should not make it absolutely positioned. Let it be in the flow. Might need a different design though? Not sure. 

Or just find a way to disable the container's `overflow: hidden` without breaking anything.

**Comment by Nyratas** [comment]
IMO we shouldn't touch Nova's base layout. There's probably a good reason (even if I couldn't find it...) for that particular `overflow: hidden`.

What are your arguments against the absolutely positioned dropdown?

**Comment by voidgraphics** [comment]
The argument is that we can not control the container's height, which means we can't prevent our popup from sticking out if it's too tall, if it's absolutely positioned.

I don't like this solution either but it might be the only way to make it work in every scenario.

**Comment by Nyratas** [comment]
The dropdown has `max-h-search overflow-y-auto`, meaning its max height is `392px`. After that, the dropdown has a scrollbar.

We could reduce it to `max-height: 140px` and position it absolutely, and it won't stick out. Here's an example: 
![image](https://user-images.githubusercontent.com/5635557/54272789-a6537e00-4584-11e9-9b99-937c48e6dfe6.png)

But the Vue.js problem remains, I have no clue how to render the dropdown outside of the field component without messing with Vue's references. I did some quick searches but all the solutions I found suggest to modify the base template, which we can't do from a simple PHP package, unless I missed something important.
